docs/tutorials/boot-tutorial.md

```diff
@@ -144,43 +161,68 @@ Now, to build the disk image, inside the disk-image folder, run:
+In this tutorial, we want to experiment with different linux kernels to examine the state of gem5's ability to boot different linux kernels. These tests use following five LTS (long term support) releases of the Linux kernel:
+
+- 4.4.186
+- 4.9.186
+- 4.14.134
+- 4.19.83
+- 5.4.49
 
-Let's use an example of kernel v5.2.3 to see how to compile the kernel.
+
+Let's use an example of kernel v5.4.49 to see how to compile the kernel.
 First, add a folder linux-configs to store linux kernel config files. The configuration files of interest are available [here](https://github.com/darchr/gem5art/blob/master/docs/linux-configs/).
-Then, we will get the linux source and checkout the required linux version (e.g. v5.2.3 in this case).
+Then, we will get the linux source and checkout the required linux version (e.g. v5.4.49 in this case).
 
 ```
 git clone https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
 mv linux linux-stable
 cd linux-stable
-git checkout v{version-no: e.g. 5.2.3}
+git checkout v{version-no: e.g. 5.4.49}
 ```
-Compile the linux kernel from its source (and an appropriate config file from linux-configs/):
+Compile the Linux kernel from its source (using an appropriate config file from linux-configs/):
 
 ```
-cp ../linux-configs/config.{version-no: e.g. 5.2.3} .config
+cp ../linux-configs/config.{version-no: e.g. 5.4.49} .config
 make -j8
-cp vmlinux vmlinux-{version-no: e.g. 5.2.3}
+cp vmlinux vmlinux-{version-no: e.g. 5.4.49}
 ```
 
 Repeat the above process for other kernel versions that we want to use in this experiment.
 
+**Note:** The above instructions are tested with `gcc 7.5.0` and the already compiled Linux binaries can be downloaded from the following links:
+
+- [vmlinux-4.4.186](http://dist.gem5.org/dist/v20-1/kernels/x86/static/vmlinux-4.4.186)
+- [vmlinux-4.9.186](http://dist.gem5.org/dist/v20-1/kernels/x86/static/vmlinux-4.9.186)
+- [vmlinux-4.14.134](http://dist.gem5.org/dist/v20-1/kernels/x86/static/vmlinux-4.14.134)
+- [vmlinux-4.19.83](http://dist.gem5.org/dist/v20-1/kernels/x86/static/vmlinux-4.19.83)
+- [vmlinux-5.4.49](http://dist.gem5.org/dist/v20-1/kernels/x86/static/vmlinux-5.4.49)
+
+
 ## gem5 run scripts
 
 Next, we need to add gem5 run scripts. We will do that in a folder named configs-boot-tests.
-Get the run script named run_exit.py from [here](https://github.com/darchr/gem5art/blob/master/docs/gem5-configs/configs-boot-tests/run_exit.py), and other system configuration files from
-[here](https://github.com/darchr/gem5art/blob/master/docs/gem5-configs/configs-boot-tests/system/).
+Get the run script named run_exit.py from [here](https://gem5.googlesource.com/public/gem5-resources/+/refs/heads/stable/src/boot-exit/configs/run_exit.py), and other system configuration files from
```

Comment:
@BobbyRBruce, do you think we should have a v20.1 branch on gem5-resources? We could have that branch last forever and have stable move to point to the same place as the "current version" branch.

This might be more overhead than it's worth, but it would make this documentation more forward-looking.

The other option is to use a specific changeset, but I don't love that solution, either.